src/traversals/recdfs.jl

```diff
@@ -0,0 +1,30 @@
+
+
+# Function to compute all simple paths between source-sink pairs
+
+# Usage: pathSet = []
+#        visited = [source]
+#        RecDFS(graph, visited, sink, pathSet)
+
+function recdfs(graph, visited, sink, pathSet)
+  nodes = LightGraphs.neighbors(graph, visited[length(visited)])
+
+  for i in nodes
+
+    if in(i, visited)
+      continue
+    end
+
+    if i == sink
+      push!(visited, i)
+      push!(pathSet, visited)
+      pop!(visited)
+    elseif !in(i, visited)
```

Comment:
infix is easier to read:
```
elseif !(i in visited)
```
Also, consider making `visited` a Set or some other data structure here, since this operation is O(n) on Vectors.